syncano_cli/execute/utils.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import json
+
+from syncano_cli.logger import get_logger
+
+LOG = get_logger('syncano-execute')
+
+
+def print_response(response):
```

Comment:
We're talking about `ScriptEndpoint`, right? Your script's returns  
```
{
    "one": 1
}
```

if executed from `syncano-cli`. Please check it Yourself.